CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * [BUGFIX] Flusher: Add `Overrides` as a dependency to prevent panics when starting with `-target=flusher`. #3151
 
 ### Mixin
+* [BUGIX] Dashboards: Fix legend showing `persistentvolumeclaim` when using `deployment_type=baremetal` for `Disk space utilization` panels. #3173
```

Comment:
> BUGIX

Oops, BUGFIX

Reply:
Sorry about that, I've cleanup after myself in this PR 
https://github.com/grafana/mimir/pull/3184